vcd/resource_vcd_vapp_vm_test.go

```diff
@@ -317,6 +317,147 @@ resource "vcd_vapp_vm" "{{.VmName2}}" {
 }
 `
 
+// TestAccVcdVmAndVAppVmWithIds tests that vApp VMs and standalone VMs can be created using the
+// `template_id` and `boot_image_id` attributes.
+// TODO: Ideally, we should refactor all tests to use `template_id` and `boot_image_id` and create a test
+// for the deprecated fields to avoid regressions.
+func TestAccVcdVmAndVAppVmWithIds(t *testing.T) {
```

Comment:
Let me know wether I should remove this test, as we have `TestAccVcdSubscribedCatalog`

Reply:
I think it's good to have